### DISCUSSION TITLE
how to launch firejail application within i3

### DISCUSSION THREAD

**Discussion by alexmaloteaux** [i3/i3#5364] [Q&A]
hi
 im not sure if this this a bug , or a special configuration is needed but i cant launch a x11 firejail app withing i3.

i know it is possible to use xinitrc/xsession to launch i3 with a firejail profile, but what i want is for i3 to be launched normally and then have default per application profile applied.

if i launched any x11 application, for instance "firejail firefox" then no sandbox is applied and i cant see the application in firejail --list. 
Console application works howerver. Can somebody provide some helps or suggestions on this ?
Best Regards

**Comment by MarrekNozka** [comment]
Could enviroment variable $I3SOCK the reason ? I run i3 in i3 via Xephyt and I must export I3SOCK correctly....
